components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1410,17 +1423,14 @@
 <nu hgrid="ne240np4">  1.1e12 </nu>
 <nu hgrid="ne0np4_arm_x8v3_lowcon"  > 8.0e-8</nu>
 <nu hgrid="ne0np4_conus_x4v1_lowcon"  > 8.0e-8</nu>
+<nu hgrid="ne0np4_northamericax4v1"  > 8.0e-8</nu>
```

Comment:
This is nit-picky, but lining up the right brackets would make the values easier to read off here (as is done above for the "uniform" resolution grids).